Source/DafnyLanguageServer/Handlers/DafnyHoverHandler.cs

```diff
@@ -259,7 +274,7 @@ class AssertionBatchIndexComparer : IComparer<AssertionBatchIndex> {
           // however, nested postconditions should be displayed
           if (errorToken is BoogieRangeToken rangeToken && !hoveringPostcondition) {
             var originalText = rangeToken.PrintOriginal();
-            deltaInformation += "  \n" + CouldProveOrNotPrefix + originalText;
+            deltaInformation += "  \n" + (token == null ? CouldProveOrNotPrefix : "Inside ") + "`" + originalText + "`";
```

Comment:
Here is as well, why can't the back-end emit the proper message? This seems like it's introducing inconsistency between IDE and CLI